### ISSUE TITLE
input hint does not pass Axe contrast ratio accessibility check

### ISSUE THREAD

**Issue by matthieu-foucault** [ericgio/react-bootstrap-typeahead#638]
### Steps to reproduce

Run an accessibility check using the Axe browser plugin, or `cypress-axe`

### Expected Behavior

No accessibility rule violation

### Actual Behavior

The input hint element fails the contrast ratio check

**Comment by matthieu-foucault** [comment]
This issue was also reported here: https://github.com/ericgio/react-bootstrap-typeahead/issues/624#issuecomment-821469510

**Comment by ericgio** [comment]
Fixed in v5.2.0
